orogen_syskit_tests.orogen

```diff
@@ -5,11 +5,25 @@ version "0.1"
 
 using_library "base-logging"
 
+import_types_from "base"
+import_types_from "orogen_syskit_testsTypes.hpp"
+typekit.export_types "/orogen_syskit_tests/TestType"
+
 # An empty component
 task_context "Empty" do
     needs_configuration
 end
 
+task_context "StaticTestType" do
```

Comment:
Let's try to be more explicit in the naming of the type and task ... They are pretty un-specific right now.

Something like `LogicalTimeType` and `LogicalTimeLoggingTest` ?